### PR TITLE
[17.0][OU-IMP] new analysis (december 2024)

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
It hasn't changed much, as the last one was done on October (https://github.com/OCA/OpenUpgrade/commit/9bde91fbf9356f10077870e815292d8083e43f8c).

I added `"sale_quotation_builder": "sale_pdf_quote_builder`" in apriori, because expectations. People having the old one expects having the new one after the migration, independently if the code has changed between the modules.

### CODE REVIEW

File: openupgrade_scripts/scripts/base_import_module/17.0.1.0/upgrade_analysis_work.txt
Comment (hbrunn) [inline openupgrade_scripts/scripts/base_import_module/17.0.1.0/upgrade_analysis_work.txt:1]: what's up with this rename?

**Review by hbrunn — CHANGES_REQUESTED** [review]
I think the rename should go into another commit than the analysis

File: openupgrade_scripts/scripts/base_import_module/17.0.1.0/upgrade_analysis_work.txt
Comment (MiquelRForgeFlow) [reply openupgrade_scripts/scripts/base_import_module/17.0.1.0/upgrade_analysis_work.txt:1]: It's an error from https://github.com/OCA/OpenUpgrade/commit/60f18195d9c9db8b620a2610cbc150a1aaae3312, that was put the file in the wrong folder.

Comment (pedrobaeza) [reply openupgrade_scripts/scripts/base_import_module/17.0.1.0/upgrade_analysis_work.txt:1]: cc @legalsylvain please take care of these things being the coordinator.

Comment (legalsylvain) [reply openupgrade_scripts/scripts/base_import_module/17.0.1.0/upgrade_analysis_work.txt:1]: Sorry. My bad.

**Comment by hbrunn** [comment]
bu still I think the addition to apriori.py should go in its own commit

**Review by hbrunn — APPROVED** [review]

**Pull request merged**